### DISCUSSION TITLE
Add way to export mass amounts of pcap data from SO2

### DISCUSSION THREAD

**Discussion by cyb3rslack3r** [Security-Onion-Solutions/securityonion#4163] [Ideas] [answered]
Would like to see a better option then stenoread to export TBs of pcap data into separate pcap files

**Comment by TOoSmOotH** [comment, marked as answer]
You would need to request that feature from the google team. Currently, stenoread is the only method to read the indexed PCAP. Indexed PCAP allows SO to pull PCAPs in a fast and reliable way. Searching over raw PCAP in the previous version of SO had many limitations and issues especially when it was capturing on busy sensors.

**Comment by defensivedepth** [comment]
@cyb3rslack3r 

What exactly is the use case here?

**Reply by cyb3rslack3r** [reply]
Exporting pcap to network tools that are standalone
